recipes/clangdev/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "3.8.1" %}
+{% set sha256 = "4cd3836dfb4b88b597e075341cae86d61c63ce3963e45c7fe6a8bf59bb382cdf" %}
+
+package:
+  name: clangdev
+  version: {{ version }}
+
+source:
+  fn: cfe-{{ version }}.src.tar.xz
+  url: http://llvm.org/releases/{{ version }}/cfe-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [(unix and py27) or (win and py<35)]
+  features:   # [win and py35]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    # Needed to unpack the source tarball
+    - m2-xz  # [py27 and win32]
+    - m2w64-xz  # [py27 and win64]
+
+    - system  # [linux and not armv7l]
+    # "compiling .pyc files" fails without this:
+    - python
+    - llvmdev =={{ version }}
+    - libxml2
+  run:
+    - system  # [linux and not armv7l]
+    - llvmdev =={{ version }}
+    - libxml2
+
+test:
+  commands:
+    - clang --version  # [unix]
```

Comment:
Should we add a test for `clang++` (admittedly it is just a symlink, but still)? Are there any additional headers or libraries we should test for?